src/Microsoft.AspNetCore.SystemWebAdapters/HttpPostedFile.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Microsoft.AspNetCore.Http;
+
+namespace System.Web;
+
+public sealed class HttpPostedFile
+{
+    public HttpPostedFile(IFormFile file) => File = file;
```

Comment:
Should this constructor be internal?

Reply:
it's only available on .NET 6 and I did that so people could test things if they wanted. I can make it internal if you have a preference (we test it, but we have internals visible to)